asv_bench/README.md

```diff
@@ -75,12 +75,14 @@ Step 2: running benchmarks with saving the results in [modin-bench@master](https
 Step 3: converting the results to html representation, which is saved in [modin-bench@gh-pages](https://github.com/modin-project/modin-bench)
   The implementation can be found in `deploy-gh-pages` job of [push.yml](https://github.com/modin-project/modin-bench/blob/master/.github/workflows/push.yml)
 
-Basic actions for 2 step:
+Basic actions for step 2:
 * setup environment variable:
   * export MODIN_TEST_DATASET=Big
   * export MODIN_CPUS=44
 * setup git client
 * prepare json file with machine description
+  * This file should be placed in the user's home directory.
+  * [Example](https://github.com/modin-project/modin-bench/blob/master/results/xeon-e5/machine.json)
```

Comment:
Example is a good thing, but link to docs describing it would be an awesome addition as well :)

Reply:
link is added.